### DISCUSSION TITLE
Very slow work on Mac with ARM chip and GraalVM question

### DISCUSSION THREAD

**Discussion by MRoose** [eclipse-birt/birt#1452] [Q&A] [answered]
Hello.
I changed work laptop on MacBook with M2 chip and install latest 4.13 release.
When opening a template, the program (birt-report-designer all-in-one) freezes for a few seconds.
Template scrolling also occurs with freezes.
The response to a click on an element from the template occurs within a few seconds.
Similar behavior on 4.12 release also.

After install, i edit birt.ini :
`-Xms2048m
-Xmx8192m`
This did not greatly affect the processing speed of the template after opening it.

Previously on Windows I used the version 4.9
I install on Mac version 4.9 for MacOS x86, but after opening template, it was outside the viewing and editing area.

I would also like to inquire about plans to replace the JRE in the distribution with compilation into a native image using GraalVM.
Thanks

**Comment by hvbtup** [comment, marked as answer]
There are no plans to use GraalVM.
And I suspect that using a native image is out of scope anyway. The reports may contain Javascript code which can call any Java class, so this is higly dynamic.
